### ISSUE TITLE
[Install] Error 404 when dowloading semantic space

### ISSUE THREAD

**Issue by jibe-b** [PeARSearch/PeARS#47]
>wget aurelieherbelot.net/pears-demo/wikiwoods.dump.bz2

returns

```
Connecting to aurelieherbelot.net (aurelieherbelot.net)|69.28.199.220|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
```

**Comment by minimalparts** [comment]
Hi @jibe-b! Thanks for trying PeARS! The problem here is that you seem to be installing the master branch... which is now very out of date, I'm afraid. Try the development branch at https://github.com/PeARSearch/PeARS/tree/development. (More instructions are on the wiki at https://github.com/PeARSearch/PeARS/wiki.) And of course, let us know if anything is broken!

**Comment by jibe-b** [comment]
Ok fine. That's a bit infortunate, though, as the landing Readme is indeed the one of the master branch.

I suggest at least indicating in the Readme of the master branch to immediately check the development branch.

**Comment by minimalparts** [comment]
Yes, you're completely right -- we should fully change the readme in master. But it does warn people to switch to development (right at the top of the readme -- perhaps not visible enough):

> Important If you are planning on trying out PeARS, head over to the development branch. The masters branch is by now very out-of-date, and the information below is too!

Sorry again about the confusion! I'll fully re-write the master readme.
